Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityBoundaryVisualizationProfileInspector.cs

```diff
@@ -94,12 +94,6 @@ public override void OnInspectorGUI()
             EditorGUILayout.Space();
             EditorGUILayout.LabelField("Boundary Visualization Options", EditorStyles.boldLabel);
             EditorGUILayout.HelpBox("Boundary visualizations can help users stay oriented and comfortable in the experience.", MessageType.Info);
-            // Boundary settings depend on the experience scale
-            if (MixedRealityToolkit.Instance.ActiveProfile.TargetExperienceScale != ExperienceScale.Room)
-            {
-                EditorGUILayout.Space();
-                EditorGUILayout.HelpBox("Boundary visualization is only supported in Room scale experiences.", MessageType.Warning);
-            }
```

Comment:
Will a future change re-introduce this warning (it seems helpful), or is this something that will be too difficult to support going forward?

Reply:
Good question. Something similar may be introduced elsewhere (ex: in the main profile when selecting the experience scale) that better describes the expected behaviors.

Reply:
Part of the intent with removing this warning is to avoid reaching across profiles for information (better separation)